src/Matrix.jl

```diff
@@ -222,32 +222,41 @@ function isone(a::MatrixElem{T}) where T <: NCRingElement
             end
          end
       end
-  end
-  return true
+   end
+   return true
 end
 
+@doc raw"""
+    is_zero_entry(M::MatrixElem{T}, i::Int, j::Int) where T <: NCRingElement
+
+Return `true` if $M_{i,j}$ is zero.
+"""
+@inline is_zero_entry(M::MatrixElem{T}, i::Int, j::Int) where T <: NCRingElement = iszero(M[i,j])
+
+@inline is_zero_entry(M::MatrixElem{T}, is::Tuple{Int, Int}) where T <: NCRingElement = is_zero_entry(M, is[1], is[2])
```

Comment:
Is this also coming from Hecke?

Reply:
Hecke imports and uses the function from Nemo for `ZZMatrx`. Furthermore, it defines a similar function `_is_zero_entry` returning a boolean and the entry itself both for `ZZMatrix` and for the general case in https://github.com/thofma/Hecke.jl/blob/8c53e50b756dc2abb1bd57b6566b65d2bce747cc/src/Sparse/Matrix.jl#L274-L283. This is only used in https://github.com/thofma/Hecke.jl/blob/8c53e50b756dc2abb1bd57b6566b65d2bce747cc/src/Sparse/Matrix.jl#L316

I would suggest either removing this function from Hecke to reduce confusion or changing the interface of the proposed AbstractAlgebra and current Nemo methods to be able to remove the Hecke functions as duplicates of upstream.

Reply:
The Hecke functions do not add a method with second argument `Tuple{Int, Int}`, so I am still confused.